tox.ini

```diff
@@ -44,13 +43,13 @@ extras =
 deps =
     xdist: pytest-xdist
     oldestdeps: minimum_dependencies
-commands_pre =
+commands_pre = 
     oldestdeps: minimum_dependencies jwst --filename {toxinidir}/requirements-min.txt
     oldestdeps: pip install -r {toxinidir}/requirements-min.txt
-    stdevdeps: pip install -r {toxinidir}/requirements-dev-st.txt -U --upgrade-strategy eager
-    devdeps: pip install -r {toxinidir}/requirements-dev-thirdparty.txt -U --upgrade-strategy eager
-    pip freeze
 commands =
+    devdeps: pip install --ignore-installed -r {toxinidir}/requirements-dev.txt -U --upgrade-strategy eager
```

Comment:
You might have  to move this one back to commands_pre because the astropy traceback looks funky and astropy dev isn't installed in devdeps

Reply:
back to the start 😅

Reply:
Not really because now we have that extra command to force numpy upgrade